src/__tests__/BoardSizeForm.test.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { shallow, mount } from 'enzyme';
+import BoardSizeForm from '../components/BoardSizeForm';
+
+describe('BoardSizeForm component', () => {
```

Comment:
I usually just see the name of the thing and lowercase for the test cases

Reply:
Do you mean you'd write `describe('board size form component' ...` ?

Reply:
`describe('BoardSizeForm')` and the test cases in lower case